app/_how-tos/synchronize-with-git.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Synchronize Insomnia content with Git
+
+products:
+- insomnia
+
+tags:
+- documents
+- collections
+- mock servers
+- environments
+- git
+
+prereqs:
+    inline:
+    - title: Git repository
+      content: |
+        Create an empty remote Git repository.
+      icon_url: /assets/icons/git.svg
+    - title: Insomnia workspace
+      content: |
+        Create a workspace in Insomnia. A workspace can be a [design document](/how-to/create-a-design-document), a [collection](), a [mock server](), or an [environment]().
+      icon_url: /assets/icons/git.svg
+
+tldr:
+    q: How can I push content from Insomnia to a Git repository?
+    a: Create an empty Git repository and an Insomnia workspace. In the workspace, click the button at the bottom of the left pane and connect the repository, then commit and push your content.
+---
+
+## 1. Select the workspace to synchronize
+
+{% navtabs %}
+
+{% navtab "Cloud Sync project" %}
+1. Click **master** at the bottom of the left pane. 
+2. Click **Switch to Git Repository**.
+{% endnavtab %}
+
+{% navtab "Local Vault project" %}
+1. Click **Not synced** at the bottom of the left pane. 
+2. Click **Connect Repository**.
+{% endnavtab %}
+
+{% endnavtabs %}
+
+## 2. Connect to the repository
+
+Select whether you want to clone the repository from GitHub, GitLab, or Git.
+
+{% capture sync %}
+1. Click **Sync**.
+{% endcapture %}
+
+{% navtabs %}
+
+{% navtab "GitHub" %}
+{% include how-tos/steps/insomnia-github.md %}
+{{ sync }}
+{% endnavtab %}
+
+{% navtab "GitLab" %}
+{% include how-tos/steps/insomnia-gitlab.md %}
+{{ sync }}
+{% endnavtab %}
+
+{% navtab "Git" %}
+{% include how-tos/steps/insomnia-git.md %}
+{{ sync }}
+{% endnavtab %}
+
+{% endnavtabs %}
+
+## 3. Create a new branch
+
+Insomnia synchronizes with the repository's default branch, but it's a good practice to make changes on a different branch.
```

Comment:
Why is it a good practice to do this? Is it like when we on the docs team open up a new branch and then open a pr from that branch to `main`?

Reply:
Yes it's just a general good practice with git to work on a branch and then merge it to the default branch